src/sagemaker/model.py

```diff
@@ -658,12 +658,7 @@ def compile(
             if target_instance_family == "ml_eia2":
                 pass
             elif target_instance_family.startswith("ml_"):
-                self.image_uri = self._compilation_image_uri(
-                    self.sagemaker_session.boto_region_name,
-                    target_instance_family,
-                    framework,
-                    framework_version,
-                )
+                self.image_uri = job_status["InferenceImage"]
```

Comment:
What if the API response returns None?

Reply:
In theory, it would never be None for this target instance family if the compilation job is successful.
Just in case that happens, I can log a warning here:
```
if self.image_uri == None:
    LOGGER.warning("InferenceImage is None for {target_instance_family}")
```
I don't want to make it an exception since the compilation job can still be successful.
What do you think?

Reply:
can you please change this to `self.image_uri = job_status.get("InferenceImage", None)`

Reply:
will do